### PR TITLE
[Integration][Datadog] Add hosts to default blueprints

### PR DESCRIPTION
**Pull request by phalbert**
# Description

```json
{
    "id": 875145983,
    "tags_by_source": {
        "Amazon Web Services": [
            "availability-zone:eu-west-1b",
            "aws_account:851725549828",
            "image:ami-0c1c30571d2dae5c9",
            "instance-type:t2.micro",
            "kernel:none",
            "name:aws-devbox-1",
            "region:eu-west-1",
            "security-group:sg-0fb806907bf6a1601"
        ],
        "Datadog": [
            "host:i-02ae0282d1c15df7c"
        ]
    },
    "aliases": [
        "i-02ae0282d1c15df7c"
    ],
    "apps": [
        "aws"
    ],
    "sources": [
        "aws"
    ],
    "name": "i-02ae0282d1c15df7c",
    "host_name": "i-02ae0282d1c15df7c",
    "aws_id": "i-02ae0282d1c15df7c",
    "aws_name": "aws-devbox-1",
    "up": true,
    "last_reported_time": 1721752690,
    "is_muted": false,
    "mute_timeout": null,
    "meta": {},
    "metrics": {
        "cpu": 0.066398375,
        "iowait": null,
        "load": null
    }
}
```

## Type of change

Please leave one option from the following and delete the rest:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New Integration (non-breaking change which adds a new integration)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Non-breaking change (fix of existing functionality that will not change current behavior)
- [ ] Documentation (added/updated documentation)

## Screenshots

Include screenshots from your environment showing how the resources of the integration will look.

## API Documentation

Provide links to the API documentation used for this integration.

### CODE REVIEW

**Review by matan84 — APPROVED** [review]
LGTM

**Review by Tankilevitch — CHANGES_REQUESTED** [review]
missing version bump and changelog

**Comment by phalbert** [comment]
> missing version bump and changelog

I have resolved this now

**Review by Tankilevitch — APPROVED** [review]

**Review by Tankilevitch — CHANGES_REQUESTED** [review]
you didn't bump the pyproject.toml

File: integrations/datadog/poetry.lock
Comment (matan84) [inline integrations/datadog/poetry.lock:1]: Why did this file change?

File: integrations/datadog/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+# Port_Ocean 0.1.12 (2024-06-25)
+
+### Features
+
+- Added the datadog host blueprint
```
Comment (matan84) [inline integrations/datadog/CHANGELOG.md:14]:
```suggestion
- Added the datadog host blueprint to the default blueprints
```

File: integrations/datadog/.port/resources/blueprints.json [hunk 2]
```diff
@@ -1,4 +1,61 @@
...
+          "description": "the host description"
+        },
+        "platform": {
+          "type": "string",
+          "title": "Platform",
+          "description": "the host platform"
+        },
+        "machine": {
+          "type": "string",
+          "title": "Machine"
```
Comment (matan84) [inline integrations/datadog/.port/resources/blueprints.json:31]: Add description

File: integrations/datadog/.port/resources/blueprints.json [hunk 3]
```diff
@@ -1,4 +1,61 @@
...
+          "description": "the Datadog agent version installed on the host"
+        },
+        "is_muted": {
+          "type": "boolean",
+          "title": "Is Host Muted?",
+          "description": "Indicates whether alerts for that specific host are temporarily suppressed."
+        },
+        "sources": {
+          "title": "Sources",
+          "type": "array"
```
Comment (matan84) [inline integrations/datadog/.port/resources/blueprints.json:50]: Add description

**Review by matan84 — CHANGES_REQUESTED** [review]
Requires a version bump in the `pyproject.toml` file

File: integrations/datadog/poetry.lock
Comment (phalbert) [reply integrations/datadog/poetry.lock:1]: any change here was not intentional. It did change while I was installing the latest ocean core in the integration

**Review by matan84 — APPROVED** [review]
LGTM

File: integrations/datadog/poetry.lock
Comment (Tankilevitch) [inline integrations/datadog/poetry.lock:1]: remove

File: integrations/datadog/.port/resources/blueprints.json [hunk 4]
```diff
@@ -1,4 +1,63 @@
...
+          "description": "Source or cloud provider associated with your host."
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {}
+  },
   {
```
Comment (Tankilevitch) [inline integrations/datadog/.port/resources/blueprints.json:61]: missing relation? from or to?

**Review by Tankilevitch — CHANGES_REQUESTED** [review]
missing pyproject.toml update

File: integrations/datadog/.port/resources/blueprints.json [hunk 5]
```diff
@@ -1,4 +1,63 @@
...
+          "description": "The CPU architecture of the host machine (e.g., amd64, x86, x64, arm, arm64)."
+        },
+        "cpuCores": {
+          "type": "number",
+          "title": "CPU Cores",
+          "description": "the host CPU cores"
+        },
+        "agent_version": {
+          "type": "string",
+          "title": "Datadog Agent Version",
```
Comment (Tankilevitch) [inline integrations/datadog/.port/resources/blueprints.json:41]: please be consistent with the properties identifiers lets use underscore, don't forget to update mapping as well

**Review by Tankilevitch — CHANGES_REQUESTED** [review]

File: integrations/datadog/.port/resources/blueprints.json (hunk 5 above)
Comment (phalbert) [reply integrations/datadog/.port/resources/blueprints.json]: alright

**Review by Tankilevitch — APPROVED** [review]
LGTM 🌊

**Pull request merged**